vnext/Shared/DevSupportManager.cpp

```diff
@@ -232,6 +235,26 @@ void DevSupportManager::StopPollingLiveReload() {
   m_cancellation_token = true;
 }
 
+void DevSupportManager::startInspector(const std::string &packagerHost, const uint16_t packagerPort) {
+#if defined(HERMES_ENABLE_DEBUGGER)
+  auto packageStatics = winrt::get_activation_factory<winrt::Windows::ApplicationModel::IPackageStatics>(
+  winrt::name_of<winrt::Windows::ApplicationModel::Package>());
```

Comment:
can we just use Package::Current() instead of dropping down to the static class?